src/eddsa.js

```diff
@@ -1,6 +1,6 @@
 import { Scalar } from "ffjavascript";
 import buildBabyJub from "./babyjub.js";
-import buildPedersenHash from "./pedersenhash.js";
+import buildPedersenHash from "./pedersenHash.js";
```

Comment:
There was also an issue with the file name of pedersenHash

Reply:
+

Reply:
Maybe because some file systems ignore case in filenames, so wrong names can work sometimes